REVIEW_GUIDELINES.md

```diff
@@ -33,7 +33,28 @@ Stable charts should not depend on charts in incubator.
 
 ## Names and Labels
 
-Resources and labels should follow some conventions. The standard resource metadata should be this:
+Resources and labels should follow some conventions.
+
+**NOTE:** 
+*The guidelines for standard labels have changed, but due to labels being immutable when using apps/v1, charts with the
+legacy labels should continue to use the old label-style, while new charts should start using the updated labels.*
```

Comment:
Do we want all charts to use new labels in the long term? If so it should be stated that it can be done along with another needed breaking change.

Reply:
I agree with comments like #9332 (review). The consensus in meetings so far has been that we shouldn't unnecessarily make breaking changes to charts, and updating to the new labels isn't a good enough reason. But if there are going to be significant enough breaking changes to a chart anyway, it could be a good time to update to the new labels in the same PR.

@davidkarlsen I just suggested a clarification in a PR on your fork against this PR's branch to : https://github.com/davidkarlsen/charts/pull/2 in case that helps 😄